Sources/Stencil/AssetsCatalogContext.swift

```diff
@@ -38,11 +38,19 @@ extension AssetsCatalogParser {
       switch entry {
       case let .group(name: name, items: items):
         return [
+          "type": "group",
           "name": name,
           "items": structure(entries: items)
         ]
+      case let .color(name: name, value: value):
+        return [
+          "type": "color",
+          "name": name,
+          "value": value
```

Comment:
Well I think `fullname` would be better here.

Only problem is, for consistency we should do the same naming for images which would then be breaking 😥